src/api/app/views/shared/_buildresult_box.html.erb

```diff
@@ -48,8 +48,8 @@
         url: '<%= url_for(:controller => controller, :action => 'rpmlint_result') %>',
         data: {
           <%= raw("'project': '#{project}',") if defined?(project) %>
-	  <%= raw("'package': '#{package}',") if defined?(package) %>
-	  <%= raw("'index': '#{index}',") if defined?(index) %>
+          <%= raw("'package': '#{package}',") if defined?(package) %>
```

Comment:
I can see that the `raw()` method was already used before you made this change but is this safe? What would the value of `package` be?